tools/reviewer/web/src/app/dashboard/reviews-panel/review/person-list/person-list.component.ts

```diff
@@ -1,97 +1,86 @@
-import { Diff } from '@/shared';
-import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 
-// The PersonListComponent is used to display 'reviewers'
-// and 'cc' list of persons
+import { FirebaseService, NotificationService } from '@/shared/services';
+import { Diff, Reviewer } from '@/shared/shell';
 
+// The PersonListComponent is used to display reviewers
 @Component({
   selector: 'app-person-list',
   templateUrl: './person-list.component.html',
   styleUrls: ['./person-list.component.scss']
 })
-export class PersonListComponent {
+export class PersonListComponent implements OnInit {
+  title: string;
   @Input() diff: Diff;
-  // Property name in Diff, e.g 'reviewers' or 'cc'.
-  // The property value is stored in propertyValue
-  @Input() property: string;
+  @Input() isNeedAttention: boolean;
   @Input() enableAddToAttention: boolean = true;
   @Input() editable: boolean = true;
-  @Output() onUpdateDiff = new EventEmitter<Diff>();
-  @Output() onRemoveProperty = new EventEmitter<string>();
-  @Output() onAddToAttentionList = new EventEmitter<string>();
+
+  constructor(
+    private firebaseService: FirebaseService,
+    private notificationService: NotificationService,
+  ) { }
+
+  ngOnInit() {
+    this.title = this.isNeedAttention ? 'Reviewers' : 'CC';
+  }
 
   // To show editable fields
   showEditableProperty = false;
 
   // Following variable is used in editing the fields
-  propertyValue: string = '';
-  persons: string[] = [];
+  reviewerEmails: string = '';
+  reviewers: string[] = [];
 
   ngOnChanges() {
     this.getPropertyValue();
   }
 
   // Get property value from the Diff
   getPropertyValue(): void {
-    switch (this.property) {
-      case 'reviewers':
-        this.persons = this.diff.getReviewersList();
-        break;
-      case 'cc':
-        this.persons = this.diff.getCcList();
-        break;
-      default:
-        throw new Error('Unsupported property');
-    }
+    this.reviewers = this.diff.getReviewerList()
+      .filter(reviewer => {
+        return reviewer.getNeedsattention() === this.isNeedAttention;
```

Comment:
`reviewer.getNeedsAttention`

Reply:
No, `getNeedsattention()` it correct

Reply:
It would be better if you could elaborate your answer with some supporting arguments rather than just declaring something. Maybe then I would be better able to see your point.

Reply:
`getNeedsattention()` is an method from proto function.
It's autogenerated. I don't choose a case of the names.
Original code:
```typescript
export class Reviewer extends jspb.Message {
  getEmail(): string;
  setEmail(value: string): void;

  getNeedsattention(): boolean;
  setNeedsattention(value: boolean): void;
```
You can see it by yourself by run 'npm run protoc` and open
`src/app/shared/shell/proto`

Reply:
bro, I didn't say that there's an error. Yeah, I said to change it as it is the simplest thing to say as it looked like the casing was incorrect. As I said It would have been better if you gave this explanation in your first response rather than declaring that the casing is correct and saved both of us time. It is not possible for a reviewer to know everything about code.